### PR TITLE
[GH-413] Picker Horizontal TextAligment

### PR DESCRIPTION
**Pull request by pictos**
### Description of Change ###

Implements #413

### Additions made ###

* Adds 

 - Adds `ITextAligment` to the `IPicker` interface
- Adds HorizontalTextAlignment property map to PickerHandler
- Adds HorizontalTextAlignment methods map to PickerHandlerfor Android and iOS
- Adds DeviceTests for initial Padding values on iOS and Android

### PR Checklist ###

- [ ] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)
- [ ] Targets a single property for a single control (or intertwined few properties)
- [ ] Adds the property to the appropriate interface
- [ ] Avoids any changes not essential to the handler property
- [ ] Adds the mapping to the PropertyMapper in the handler
- [ ] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [ ] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [ ] Tags ported renderer methods with [PortHandler]
- [ ] Adds an example of the property to the sample project (MainPage)
- [ ] Adds the property to the stub class
- [ ] Implements basic property tests in DeviceTests

#### Does this PR touch anything that might effect accessibility?
- [ ] APIs that modify focusability?
- [ ] APIs that modify any text property on a control?
- [ ] Does this PR modify view nesting or view arragement in anyway?
- [ ] Is there the smallest possibility that your PR will change accessibility? 
- [ ] I'm not sure, please help me

If any of the above checkboxes apply to your PR then the PR will need to provide testing to demonstrate that accessibility still works.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -2,5 +2,8 @@
   "msbuild-sdks": {
     "MSBuild.Sdk.Extras": "3.0.23",
     "Microsoft.Build.NoTargets": "2.0.1"
+  },
+  "sdk": {
```
Comment (jsuarezruiz) [inline global.json:6]: Could you revert this change?

**Review by jsuarezruiz — APPROVED** [review]

File: global.json (hunk 1 above)
Comment (pictos) [reply global.json]: done

**Review by rachelkang — APPROVED** [review]
Looks great to me! Thanks Pedro :)
